src/main/java/hudson/maven/MavenEmbedderUtils.java

```diff
@@ -43,7 +43,6 @@
 import org.codehaus.plexus.classworlds.ClassWorld;
 import org.codehaus.plexus.classworlds.realm.ClassRealm;
 import org.codehaus.plexus.util.IOUtil;
-import org.eclipse.sisu.BeanScanning;
```

Comment:
Seems so. BTW it raises a valid compatibility concern if this or other similar class is somehow being used in plugins through dependencies